### PR TITLE
Fixes #11687: Create a first implementation of multiversion technique

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/11687

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

File: techniques/fileDistribution/fileTemplate/1.0/metadata.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
```
Comment (fanf) [inline techniques/fileDistribution/fileTemplate/1.0/metadata.xml]: We also need to specify that the technique support the generation of multiple policy without merge. 
For now, the xml element for that is: 
  <POLICYGENERATION>multi</POLICYGENERATION>

(and a default value of "merge" in place of multi, if the tag is missing, for compatibility)

Comment (peckpeck) [reply techniques/fileDistribution/fileTemplate/1.0/metadata.xml]: I'd replace "multi" with "separate"

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

**Comment by fanf** [comment]
Works fine with #1780. Merging it!

File: techniques/fileDistribution/fileTemplate/1.0/fileTemplate.ps1.st [hunk 2]
```diff
@@ -1,4 +1,4 @@
-function fileTemplate {
+function fileTemplate_RudderUniqueID {
```
Comment (ncharles) [inline techniques/fileDistribution/fileTemplate/1.0/fileTemplate.ps1.st]: metadata has not been accordingly changed :/

**Pull request merged**